app/source/js/viewmodels/ServiceProfessionalService.js

```diff
@@ -80,7 +84,7 @@ function ServiceProfessionalServiceViewModel(app) {
             });
         }
         
-        if (!this.isSelectionMode()) {
+        if (!this.isSelectionMode() || !this.loadEmptyPricingTypes()) {
```

Comment:
Nice catch!

I agree. I dropped the check on `isSelectionMode` in that block and am checking the positive of `loadEmtpyPricingTypes`. 

I've tested this

- As a professional managing own services and add-ons
- As a professional booking services for a client
- As a client booking services from a professional

I've pushed my changes. Let me know if there's anything else I should fix.